### PR TITLE
feat: add exlcuded methods option

### PR DESCRIPTION
**Pull request by douglaseggleton**
# Description

Adds the option to exclude payment methods.

Fixes # https://gr4vy.atlassian.net/browse/TA-5969

## Contribution guidelines

For contribution guidelines, styleguide, and other helpful information please
see the `CONTRIBUTING.md` file in the root of this project.

### CODE REVIEW

File: packages/embed-react/README.md [hunk 1]
```diff
@@ -88,6 +88,7 @@ The options for this integration are as follows.
...
+| `excludedMethods`         | `false`     | An optional list of methods to be excluded.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
```
Comment (gryevns) [inline packages/embed-react/README.md:91]: Is the default `false` here or should it be `null` or `[]`?

Comment (douglaseggleton) [inline packages/embed-react/README.md:91]:
```suggestion
| `excludedMethods`         | `null`     | An optional list of methods to be excluded.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
```

File: packages/embed/README.md [hunk 2]
```diff
@@ -99,6 +99,7 @@ The options for this integration are as follows.
...
+| `excludedMethods`         | `false`      | An optional list of methods to be excluded.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
```
Comment (douglaseggleton) [inline packages/embed/README.md:102]:
```suggestion
| `excludedMethods`         | `null`      | An optional list of methods to be excluded.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
```

**Review by gryevns — APPROVED** [review]

**Comment by gr4vy-code** [comment]
:rocket: PR was released in [`v2.25.0`](https://github.com/gr4vy/gr4vy-embed/releases/tag/v2.25.0) :rocket:

**Pull request merged**